Yank/commands/script.py

```diff
@@ -25,14 +25,17 @@
 YANK script
 
 Usage:
-  yank script (-y=FILEPATH | --yaml=FILEPATH)
+  yank script (-y=FILEPATH | --yaml=FILEPATH) [--platform=PLATFORM]
```

Comment:
Oh, I see your confusion. This is not redundant. There is no way to specify the platform name in they YAML script.